app/routes/__frontend/resources/virtual-coffee-handbook/guides-to-virtual-coffee/slack-channels-guide.mdx

```diff
@@ -0,0 +1,383 @@
+
+Do you want to discuss or have any questions about full-stack? Then you're looking at the right channel!
+
+### #dev-ml-data-ai
+
+Anything about machine learning (ML), statistics, linguistics, and AI is happening in this channel.
+
+### #dev-vue-crew
+
+The channel to discuss and learn everything about VueJS.
+
+### #dev-wordpress
+
+The place to discuss all things WordPress!
+
+### #devrel
+
+A channel for all things Developer Relations! Contents, travel, community, and more!
+
+## E
+
+### #entertainment
+
+The floodgates are open to discuss all of our beloved entertainment. Music, TV shows, movies, manga/anime, comics, Marvel, etc. As always, the <Link to="/code-of-conduct">Code of Conduct</Link> applies. No NSFW content.
+
+### #external-events
+
+Do you know some non-VC-affiliated tech events and want to share them with us? Drop them here!
+
+## F
+
+### #food
+
+Do you want to share delicious foods you know, your Instagram-like (or not) food pics, or ask and share recipes? We welcome all foodies here!
+
+### #foreign-languages-and-culture
+
+This channel covers all languages, including English. In addition, use this as a space to share different aspects of your various cultures, like regional holidays, family traditions, and anything you find interesting or fun!
+
+## G
+
+### #game-night
+
+Want to game with other Virtual Coffee members? Look no further! It can be anything from a fun website, Catan Online, or even something on Steam.
+
+### #game-result
+
+Did you nail the latest Noodle, Wordle, Hurdle, or Zoodle?!? Go you! 🎉 Share those results here, threading conversations as appropriate.
+
+### #general
+
+This is a place to post anything related to tech in general and extensions of Virtual Coffee's conversations.
+
+#### 🔁 Regular Posts in This Channel
+
+- Notes from weekly Coffees
+- Wednesday’s midweek check-in
+- Friday’s gratitude
+
+### #goals-and-wins
+
+Share your goals for the week and celebrate your wins in this channel!
+
+### #gratitude
+
+A channel dedicated to shoutouts & kind words. Spread the love 💖!
+
+## H
+
+### #happiness
+
+Just a place to post things, ANYTHING, that makes you happy!
+
+### #health-and-fitness
+
+Because we also need to think of our health! This channel is a place to share anything related to health and sports/workouts that you do.
+
+### #heavy
+
+It's a safe place to share anything heavy without judgment. Ask for support or advice, or know you're being heard 💙.
+
+### #help-and-pairing
+
+Do you have any questions about your code? Or do you need someone to help you by pairing? You can ask for it here!
+
+This channel is not limited to throwing questions or asking for help with code. You can also ask for general opinions about your project, ask for resources, help fill out surveys, or any other tech-related help without judgment!
+
+The list of members' <Link to="/resources/virtual-coffee-handbook/guides-to-virtual-coffee/glossary#office-hours">office hours</Link> is pinned in this channel.
+
```

Comment:
Link to the guide to asking questions about code here?

Reply:
We don't have it for now . But we can do that because it make sense to later pin the asking questions about code here.